python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -924,6 +936,15 @@ def _getattribute(self, name):
 
     return type(name, (GPIProxyObject,), d)
 
+_metaClassHolder = None
+def _getMetaClass():
+    global _metaClassHolder
+    if _metaClassHolder is None:
+        from Ganga.GPIDev.Base.Objects import ObjectMetaclass
+        _metaClassHolder = ObjectMetaclass
+    return _metaClassHolder
```

Comment:
Returns the ObjectMetaClass for a comparison (I don't know why the comparison is made I just do what the code used to do)
I had a problem just directly importing ObjectMetaClass in the past, plus wanted to avoid excessive calls to import unless this is safe to do?